python-lib/amazon_translation_api_formatting.py

```diff
@@ -123,6 +123,11 @@ def format_row(self, row: Dict) -> Dict:
     def format_df(self, df: pd.DataFrame) -> pd.DataFrame:
         logging.info("Formatting API results...")
         df = df.apply(func=self.format_row, axis=1)
+        # In case of an empty dataset add empty columns to maintain consistent output schema
+        if len(df) == 0:
```

Comment:
Use pandas' builtin `.empty` property ?
```suggestion
        if df.empty:
```